module-1/lab-api-scavenger-game/your-code/challenge_1.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
Good job Ivan!! Well done!! You can do it in less code lines. Normalizing the JSON, creating a dataframe and getting the unique values of a dataframe column.